app/host/transient_tasks.py

```diff
@@ -61,6 +62,90 @@ def _run_process(self, transient):
         return status_message
 
 
+class MWEBV_Transient(TransientTaskRunner):
+    """
+    TaskRunner to run get Milky Way E(B-V) values at the transient location.
+    """
+
+    def _prerequisites(self):
+        """
+        Only prerequisite is that the host match task is not processed.
+        """
+        return {"Transient MWEBV": "not processed"}
+
+    @property
+    def task_name(self):
+        """
+        Task status to be altered is host match.
+        """
+        return "Transient MWEBV"
+
+    def _failed_status_message(self):
+        """
+        Failed status - not sure why this would ever fail so just a placeholder.
+        """
+        return "failed"
+
+    def _run_process(self, transient):
+        """
+        Run the E(B-V) script.
+        """
+        mwebv = get_dust_maps(transient.sky_coord)
+
+        if mwebv is not None:
+            transient.milkyway_dust_reddening = mwebv
+            transient.save()
+            status_message = "processed"
+        else:
+            status_message = "no transient MWEBV"
```

Comment:
codecov says the else branch is not covered by at test, if this is easy please add a test the goes down the else branch, not required though :-)